### PR TITLE
Alias Beberlei\Metrics\Collector\Collector to the default collector to support autowiring

### PR DESCRIPTION
**Pull request by Tobion**
This way you can typehint the interface and get the default collector injected automatically.

### CODE REVIEW

File: src/Beberlei/Bundle/MetricsBundle/DependencyInjection/BeberleiMetricsExtension.php [hunk 1]
```diff
@@ -25,13 +26,14 @@ public function load(array $configs, ContainerBuilder $container)
...
-            $container->setAlias('beberlei_metrics.collector', 'beberlei_metrics.collector.'.$config['default']);
-        } else {
+        if ($config['default'] && $container->hasDefinition('beberlei_metrics.collector.'.$config['default'])) {
+            $name = $config['default'];
+        } elseif (1 !== count($config['collectors'])) {
             throw new \LogicException('You should select a default collector');
         }
+
+        $container->setAlias('beberlei_metrics.collector', 'beberlei_metrics.collector.'.$name);
+        $container->setAlias(Collector::class, 'beberlei_metrics.collector.'.$name);
```
Comment (stof) [inline src/Beberlei/Bundle/MetricsBundle/DependencyInjection/BeberleiMetricsExtension.php]: The autowiring alias should rather point to `beberlei_metrics.collector` itself

Comment (Tobion) [reply src/Beberlei/Bundle/MetricsBundle/DependencyInjection/BeberleiMetricsExtension.php]: done

**Comment by lyrixx** [comment]
It took time, but here we go, this is in now. Thank you very much @Tobion.

**Comment by lyrixx** [comment]
Released as v2.8.0

**Pull request merged**